docs/src/guides/typo3/deploy/configure.md

```diff
@@ -29,6 +29,152 @@ If a service stores persistent data then it will also have a `disk` key, which s
 
 ## Application container: `.platform.app.yaml`
 
-{{< guides/config-app template="typo3" >}}
```

Comment:
Please don't inline this file.  We want it pulled from the template so that it's always up to date.